src/gw_large_cell_gamma.F

```diff
@@ -1019,6 +1037,14 @@ SUBROUTINE compute_W_MIC(bs_env, qs_env, mat_chi_Gamma_tau, fm_W_MIC_time)
                ! Fourier trafo from W_PQ^MIC(iω_j) to W_PQ^MIC(iτ)
                CALL Fourier_transform_w_to_t(bs_env, fm_W_MIC_time, bs_env%fm_W_MIC_freq, j_w)
 
+               ! ! DEBUG : print the W_MIC matrices for all frequencies
+               ! PRINT *, "Frequency index", j_w, "Frequency", bs_env%imag_freq_points(j_w)
+               ! ALLOCATE(dummy_array(1))
```

Comment:
Wrap this part into
```
IF (.FALSE.) THEN
BLOCK
TYPE(cp_fm_type), ALLOCATABLE, DIMENSION(:) :: dummy_array
...
END BLOCK
END IF```
In this way, everything is tied together and the compiler can check your debug code

Reply:
Thank you for spotting those, the lines were unnecessary and as such were deleted.